docs/architecture/adr-001-autopilot-oci-basic-auth-support.md

```diff
@@ -110,6 +110,39 @@ secretRef:
   name: artifacts-token-based-auth
 ```
 
+#### Expected workflow
+
+This feature supports pulling both the k0s binary and the k0s air-gap bundle.
```

Comment:
Downloading executables raises the question of how to manage the executable bit of the downloaded files.